services/cms/src/components/breadcrumbs/EditorBreadcrumbs.tsx

```diff
@@ -0,0 +1,65 @@
+import { useRouter } from "next/router"
+import React from "react"
+import { useQuery } from "react-query"
+
+import { fetchPageInfo } from "../../services/backend/pages"
+import Breadcrumbs from "../../shared-module/components/Breadcrumbs"
+import BreakFromCentered from "../../shared-module/components/Centering/BreakFromCentered"
+import ErrorBanner from "../../shared-module/components/ErrorBanner"
+import Spinner from "../../shared-module/components/Spinner"
+
+const EditorBreadcrumbs: React.FC = () => {
+  const router = useRouter()
+
+  const pageId = router.asPath.split("/")[2]
+  const data = useQuery(`page-with-id-${pageId}`, () => {
+    if (!pageId) {
+      return null
+    }
+    return fetchPageInfo(pageId)
+  })
+
+  if (!data) {
+    return (
+      <BreakFromCentered sidebar={false}>
```

Comment:
In the CMS, there is a sidebar. One easy way to get the correct sidebar props is to use:

```suggestion
      <BreakFromCentered {...breakFromCenteredProps}>
```